### ISSUE TITLE
VidCoder Burned Subtitles Misaligned

### ISSUE THREAD

**Issue by Hozonkai** [RandomEngy/VidCoder#916]
I just updated VidCoder, but when I burned my subtitles in, they're small and misaligned to the upper right corner.

**Comment by RandomEngy** [comment]
Try it with HandBrake. VidCoder uses HandBrake for the core encoding engine, so if the issue reproduces there it's an issue with their code and you can ask them for help. If it doesn't reproduce with HandBrake, we know it's a VidCoder issue.

Equivalent versions:
VidCoder Stable ↔ [HandBrake Stable](https://handbrake.fr/downloads.php)
VidCoder Beta ↔ [HandBrake Nightly](https://handbrake.fr/nightly.php)

**Comment by Hozonkai** [comment]
After burning on Handbrake, the subtitles won't even show up at all. They would always work on VidCoder, which is what I would prefer to use. It's an MVK video btw

**Comment by RandomEngy** [comment]
Yeah, I'm just suggesting to use it as a debugging tool. They own the core encoding engine, so any subtitle issues would need to be fixed in their code. In almost all cases when there is a difference between VidCoder and HandBrake it's due to differences in the encoding settings. If we can reproduce the issue in HandBrake we can submit a bug report to them and the fix will be picked up in VidCoder. If you post the encode log for each I can help identify if there are any differences in the settings.

**Comment by Hozonkai** [comment]
Tried it again on Handbrake, but I got the same alignment problem. How do I send a bug report?

**Comment by RandomEngy** [comment]
https://github.com/HandBrake/HandBrake/issues/new?assignees=&labels=bug&template=bug_report.yml
